src/main/java/com/github/manolo8/darkbot/gui/players/PlayerRenderer.java

```diff
@@ -15,7 +15,7 @@
 
 public class PlayerRenderer extends JPanel implements ListCellRenderer<PlayerInfo> {
 
-    private final JLabel playername = new JLabel();
+    private final JLabel playerName = new JLabel();
```

Comment:
```suggestion
    private final JLabel playername = new JLabel();
```
keep this as-is, playername is used as a single word in this case, just like username